source/config/__init__.py

```diff
@@ -64,7 +64,13 @@ def isInstalledCopy():
 
 def getInstalledUserConfigPath():
 	try:
-		return os.path.join(shlobj.SHGetFolderPath(0, shlobj.CSIDL_APPDATA), "nvda")
+		k = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, NVDA_REGKEY)
+		configInLocalAppData = bool(_winreg.QueryValueEx(k, u"configInLocalAppData")[0])

```

Comment:
It's not clear to me how to read your comment here, I am assuming you meant "I honestly do not see why we **shouldn't** hard code here" (difference in bold). If I'm mistaken, I apologise.

The main reasons I can give off the top of my head for replacing duplicate hardcoded values with references to a constant are:
1. To reduce the cost of updating the value of that constant, it only needs to be changed in one place.
2. Reduces the chance of having non matching values across the code base. A constant will more likely result in a error from python, whereas a misspelling of "configInLocalAppdata" will result in incorrect behaviour.
3. It can be given a descriptive name that differs from its value.
4. All usages of the value make it easy refer back to where the value is defined making it easier to find the documentation for the value.

While it would be nice to have consistency across the code base here, we do have to draw a line somewhere when it comes to tidying the existing code. Perhaps I should not have referred you to `NVDA_REGKEY` as an example since the constant is not actually used from `installer.py`. If you wish to tidy this up as well, please do so.